process-automation/latest/develop/index.md

```diff
@@ -608,101 +608,25 @@ Restart the application and you will see the custom columns in the process list
-        "providers": "ALL",
-        "appName": "invoice-approval-application",
-        "application": {
-            "name": "Alfresco ADF Application"
-        },
-
-    "authType": "OAUTH",
-    "oauth2": {
-        "host": "https://{hostname}/auth/realms/alfresco",
-        "clientId": "invoice-approval-application",
-        "scope": "openid",
-        "secret": "",
-        "implicitFlow": true,
-        "silentLogin": true,
-        "redirectSilentIframeUri": "{protocol}//{hostname}{:port}/assets/silent-refresh.html",
-        "redirectUri": "/",
-        "redirectUriLogout": "/"
-    },
-    ```
-
-5. Add the property `"alfresco-deployed-apps"` to the `app.config.json` with the name of the deployed application you are developing an interface for. For example, `"alfresco-deployed-apps": [{"name":"invoice-approval-application"}]`.
-
-6. To develop the interface locally, CORS will need to be bypassed. Edit the `proxy.conf.js` replacing the `{hostname}` and `{realm}` with those relevant to your environment and realm:
-
-    ```js
-    module.exports = {
-        "/auth/admin/realms/{realm}": {
-            "target": "https://{hostname}",
-            "secure": false,
-            "pathRewrite": {
-                "^/auth/admin/realms/{realm}": ""
-            },
-            "changeOrigin": true,
-            "logLevel": "debug"
-        },
-
-        "/auth/realms/...": {
-            "target": "https://{hostname}",
-            "secure": false,
-            "pathRewrite": {
-                "^/auth/realms/{realm}": ""
-            },
-            "changeOrigin": true,
-            "logLevel": "debug"
-        },
-
-        "/": {
-            "target": "https://{hostname}",
-            "secure": false,
-            "changeOrigin": true,
-            "logLevel": "debug"
-        },
-
-        "/alfresco": {
-            "target": "https://{hostname}",
-            "secure": false,
-            "changeOrigin": true
-        }
-    }
-    ```
+5. Once your custom UI is ready to be deployed, select **Upload**.
 
-### Develop {#custom-develop}
+### Develop a custom user interface
 
-There are a set of [content components](https://www.alfresco.com/abn/adf/docs/content-services/){:target="_blank"} and a set of [process components](https://www.alfresco.com/abn/adf/docs/process-services-cloud/){:target="_blank"} that can be used to develop the custom user interface with. The content components can only be used if you selected the option **Content and Process Services with Activiti** when generating the application.
+Custom UIs are developed using nx generators.
 
-> **Note**: Process Automation uses the Process Services **Cloud** components.
+The downloaded default UI ZIP file includes a file with detailed instructions on creating custom UI pages and plugins. To see the instructions, refer to **plugins-generators.md** file within the downloaded ZIP package.
 
 ### Upload {#custom-upload}
```

Comment:
We can rewrite this point, and describe that user can just upload a zip file in modeling app using upload button

![Screenshot 2024-02-07 at 12 33 04](https://github.com/Alfresco/docs-alfresco/assets/13885344/5f91c4bd-1285-4da4-b01f-721210ac2511)